### DISCUSSION TITLE
Specify Server Name for Couchbase Node

### DISCUSSION THREAD

**Discussion by SPConlin** [testcontainers/testcontainers-java#4716] [Q&A]
Is there a way  to specify the node/server name to be used for a Couchbase container launched via TestContainers? 

I have recently noticed that the Couchbase server/node is named with the Docker IP when created via TestContainers but defaults to "127.0.0.1" if configured manually. I would like to be able to not use the Docker IP for the name.

**Comment by daschl** [comment]
@SPConlin I'd like to understand the use case more. We are using alternate addresses (couchbase feature) to map IPs from inside to outside the docker container, so I'm not sure if changing the hostname makes sense in this environment.

**Reply by SPConlin** [reply]
The environment in use includes multiple systems and applications interacting. I ran into this particular issue while working with Couchbase in Docker, a local application, and VMware system serving as a node for the application. Since not all of the machines are on the Docker network, I need to have the Couchbase Node to have either 127.0.0.1 or the IP of the host so that it reports a useable host name/IP
